src/modules/dashboard/components/ActionsPage/DetailsWidget/DetailsWidget.tsx

```diff
@@ -75,6 +75,14 @@ const DetailsWidget = ({
   const Amount = () => values?.amount as ReactElement;
   const Symbol = () => values?.tokenSymbol as ReactElement;
 
+  // @TODO We need to find a better way of handling for which actions we want display which fields
+  const showFromDomain =
+    !ColonyActions.MintTokens && values?.fromDomain && showFullDetails;
+  const showToDomain =
+    !ColonyActions.MintTokens &&
+    (values?.toDomain || recipient) &&
+    showFullDetails;
+
```

Comment:
This needs to be refactored in a static mapping as I did for the other stuff that is dependent on different actions or event types.